store/extensions/G-Hypertranslate/extension.json

```
@@ -0,0 +1,39 @@
+{
+  "title": "G-Hypertranslate",
+  "description": "Play translation telephone using Google Translate - https://www.ravbug.com/hypertranslate/",
+  "authors": [{
+    "name": "Alynva",
+    "hotel": ".com.br",
+    "username": "Alynva",
+    "discord": "Alynva#0001",
+    "twitter": "Alynva"
+  }],
+  "version": "0.1",
+  "categories": ["Fun"],
+
+  "source": "https://github.com/Alynva/G-Hypertranslate",
+  "readme": "https://github.com/Alynva/G-Hypertranslate#readme",
+  "releases": "https://github.com/Alynva/G-Hypertranslate/releases",
+
+  "stable": true,
+
+  "framework": {
+    "name": "G-Node",
+    "version": "0.2.9"
+  },
+
+  "language": "node",
```

The language has to be `Node.js` instead of `node`

```suggestion
  "language": "Node.js",
```

https://github.com/sirjonasxx/G-ExtensionStore/blob/repo/1.5.2/store/config.json#L82